test/array_ref.cpp

```diff
@@ -87,8 +104,8 @@ TEST(array_ref_incompatible_shape) {
   }
 
   {
-    array_ref<int, nda::shape<nda::fixed_dim<10, nda::dynamic>,
-                              nda::fixed_dim<2, nda::dynamic>>> dst;
+    array_ref<int, nda::shape<nda::fixed_dim<10, nda::dynamic>, nda::fixed_dim<2, nda::dynamic>>>
```

Comment:
Ugly - want me to revert this?

Reply:
I agree but I wouldn't bother if it's what clang-format will do every time.